source/actions/show-menu.js

```diff
@@ -0,0 +1,44 @@
+/* eslint consistent-return: off   */
+const chalk = require('chalk');
+const enquirer = require('enquirer');
+
+const {
+  randomDesktop,
+  getImages,
+  packExe,
+  showSettings,
+} = require('../actions');
+
+const { MENU_OPTIONS } = require('../constants');
+
+/* Action that pack the app into one single Windows executable file */
+module.exports = async function (args, options, logger) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Async function has a complexity of 6. Maximum allowed is 4.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=4814399)